### ISSUE TITLE
Install HPG-Variant

### ISSUE THREAD

**Issue by alanorth** [ilri/hpc-environment-modules#7]
Has a few different build requirements, but should be doable.

https://github.com/opencb/hpg-variant

**Comment by oguya** [comment]
unable to build. Raised a bug on hpg-variant repo: https://github.com/opencb/hpg-variant/issues/16

**Comment by oguya** [comment]
I got some useful tips from the conversation on the issue. So an easier way of building hpg-variant is to clone the repo, checkout to v1.0 tag & update the submodules.

```
scl enable devtoolset-2 bash
git clone https://github.com/opencb/hpg-variant.git
git submodule update --init
scons -j4
```

**Comment by oguya** [comment]
Fixed in commit: 8e55058804da142e59d301021623c86f42fb53bd

**Comment by alanorth** [comment]
So basically, they don't maintain a stable master. ;)

On Thu, Apr 2, 2015 at 6:21 AM, James Oguya notifications@github.com
wrote:

> Closed #7 https://github.com/ilri/hpc-environment-modules/issues/7.
> 
> ## 
> 
> Reply to this email directly or view it on GitHub
> https://github.com/ilri/hpc-environment-modules/issues/7#event-271685469
> .

## 

Alan Orth
alan.orth@gmail.com
https://alaninkenya.org
https://mjanja.ch
"In heaven all the interesting people are missing." -Friedrich Nietzsche
GPG public key ID: 0x8cb0d0acb5cd81ec209c6cdfbd1a0e09c2f836c0
